lntest/miner/miner.go

```diff
@@ -314,6 +314,34 @@ func (h *HarnessMiner) AssertTxInMempool(txid chainhash.Hash) *wire.MsgTx {
 	return h.GetRawTransaction(txid).MsgTx()
 }
 
+// AssertTxnsNotInMempool asserts the given txns are not found in the mempool.
+// It assumes the mempool is not empty.
+func (h *HarnessMiner) AssertTxnsNotInMempool(txids []chainhash.Hash) {
+	err := wait.NoError(func() error {
+		// We require the RPC call to be succeeded and won't wait for
+		// it as it's an unexpected behavior.
```

Comment:
nit: not sure I understand that comment

Reply:
Seems to be copy/paste, appears in a couple of other places. I interpret it as: "this will call `ht.t.Fail()` if it doesn't succeed, which will abort the test, so we can continue below assuming it didn't fail".